Composer/packages/adaptive-form/src/components/fields/AdditionalField.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import { FieldProps, FieldWidget } from '@bfc/extension-client';
+
+import { FieldLabel } from '../FieldLabel';
+import { schemaField } from '../SchemaField';
+
+interface AdditionalFieldProps extends FieldProps {
+  name: string;
+  field: FieldWidget;
+  helpLink?: string;
+  description?: string;
+  label?: string | false;
+  required?: boolean;
+}
+
+export const AdditionalField: React.FC<AdditionalFieldProps> = (props) => {
+  const { id, depth, description, field: Field, helpLink, label, required } = props;
+
+  return (
+    <div css={schemaField.container(depth)}>
```

Comment:
Is `depth` used anywhere currently? Should we remove it?

Reply:
Depth is really only used for padding the root level properties. I'd be alright with removing it.